symplyphysics/laws/electricity/circuits/resistivity_of_parallel_resistors.py

```diff
@@ -21,6 +25,57 @@
 law = Eq(parallel_resistance, 1 / SumArray(inv_resistances), evaluate=False)
```

Comment:
I suggest to rename `inv_resistances` to `conductances`. This way we will justify the usage of conductance law in the proof. And also link together resistance and conductance laws.